app/views/unattended/provisioning_templates/snippet/preseed_netplan_setup.erb

```diff
@@ -34,8 +34,8 @@ oses:
                   :interface => bond,
                   :subnet => bond.subnet,
                   :subnet6 => bond.subnet6,
-                  :dhcp => bond.subnet.nil? ? false : bond.subnet.dhcp_boot_mode?,
-                  :dhcp6 => bond.subnet6.nil? ? false : bond.subnet6.dhcp_boot_mode? }) -%>
+                  :dhcp => bond.subnet&.dhcp_boot_mode?,
+                  :dhcp6 => bond.subnet&.dhcp_boot_mode? }) -%>
```

Comment:
subnet6 IMHO
```suggestion
                  :dhcp6 => bond.subnet6&.dhcp_boot_mode? }) -%>
```

Reply:
Funny, I just reverted your commit and didn't look at the results. Was there a previous bug here?

Reply:
It's also been quite hard to see due to the repeating pattern, LGTM now!